### PR TITLE
[TEST] [e2e] Adding tests for basic promotion CRUD

### PR DESCRIPTION
**Pull request by yellowee**
It covers:
- [TC SALEOR_97 Create promotion](https://saleor.testmo.net/repositories/1?group_id=3285&case_id=16201)
- [TC SALEOR_98 Update promotion](https://saleor.testmo.net/repositories/1?group_id=3285&case_id=16209)
- [TC SALEOR_99 Delete promotion](https://saleor.testmo.net/repositories/1?group_id=3285&case_id=16216)

_Please note: I will cover Test cases for promotion rules in the next PR_

## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

### CODE REVIEW

File: playwright/pages/dialogs/addPromotionRuleDialog.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        this.page = page;
+    }
+    async typePromotionRuleDescription(description: string) {
+        await this.ruleDescriptionInput.fill(description);
+    }
+    async clickChannelsDropdown() {
+        await this.ruleChannelDropdown.click();
+    }
+    async selectSingleChannel(channel = "PLN") {
+        await this.selectOption.filter({ hasText: `Channel-${channel}` }).click();
```
Comment (poulch) [inline playwright/pages/dialogs/addPromotionRuleDialog.ts:33]: question: `selectOption` exists only when the dropdown is open, how does it work that you click on it before open dropdown?
Also I see that method `selectSingleChannel` is not used

File: playwright/pages/dialogs/addPromotionRuleDialog.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        await this.addRuleConditionSection.waitFor({
+            state: "visible",
+            timeout: 10000,
+        });
+    }
+    async selectPercentageRewardValueType() {
+        await this.percentageRewardValueTypeOption.click();
+        // await this.percentageRewardValueTypeOption.getByRole('checkbox', { 'checked': true }).waitFor({ state: 'visible' });
+        await expect(this.percentageRewardValueTypeOption).toBeChecked();}
+    async selectFixedRewardValueType() {
```
Comment (poulch) [inline playwright/pages/dialogs/addPromotionRuleDialog.ts:56]:
```suggestion
        await expect(this.percentageRewardValueTypeOption).toBeChecked();}
        
    async selectFixedRewardValueType() {
```

File: playwright/pages/dialogs/addPromotionRuleDialog.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        await expect(this.fixedRewardValueTypeOption).toBeChecked();
+    }
+    async selectRewardType(type: string) {
+        await this.rewardTypeSelect.click();
+        await this.page.getByRole("listbox").waitFor({
+            state: "visible",
+            timeout: 10000,
+        });
+        // await this.page.getByTestId("select-option").getByText(type).click();
+        await this.page.getByTestId("select-option").selectOption(type);
```
Comment (poulch) [inline playwright/pages/dialogs/addPromotionRuleDialog.ts:67]: question: why not use `this.selectOption` here?

File: playwright/pages/dialogs/addPromotionRuleDialog.ts (hunk 1 above)
Comment (yellowee) [reply playwright/pages/dialogs/addPromotionRuleDialog.ts]: @poulch I've removed `addPromotionRuleDialog.ts` file for now as it is not needed until I create tests for promotion rules (will be added back in the next PR).

File: playwright/pages/dialogs/addPromotionRuleDialog.ts (hunk 2 above)
Comment (yellowee) [reply playwright/pages/dialogs/addPromotionRuleDialog.ts]: @poulch I've removed `addPromotionRuleDialog.ts` file for now as it is not needed until I create tests for promotion rules (will be added back in the next PR).

File: playwright/pages/dialogs/addPromotionRuleDialog.ts (hunk 3 above)
Comment (yellowee) [reply playwright/pages/dialogs/addPromotionRuleDialog.ts]: @poulch I've removed `addPromotionRuleDialog.ts` file for now as it is not needed until I create tests for promotion rules (will be added back in the next PR).

**Review by poulch — APPROVED** [review]

**Review by szczecha — APPROVED** [review]

**Pull request merged**